jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -234,15 +238,20 @@ public boolean clearCurrentSelection( CellPreviewEvent<ProcessInstanceSummary> e
     @Override
     public void initColumns( ExtendedPagedTable<ProcessInstanceSummary> extendedPagedTable ) {
         final ColumnMeta checkColumnMeta = initChecksColumn();
-
-        actionsColumn = initActionsColumn();
+        Column<ProcessInstanceSummary, ?> actionsColumn = initActionsColumn();
+        Column<ProcessInstanceSummary, ?> errorCountColumn = initErrorCountColumn();
+        
+        ignoreSelectionColumns.add(actionsColumn);
+        ignoreSelectionColumns.add(errorCountColumn);
 
         final List<ColumnMeta<ProcessInstanceSummary>> columnMetas = new ArrayList<ColumnMeta<ProcessInstanceSummary>>();
 
         columnMetas.add(checkColumnMeta);
         columnMetas.add(new ColumnMeta<>(createTextColumn(COLUMN_PROCESS_INSTANCE_ID,
                                                           process -> String.valueOf(process.getProcessInstanceId())),
                                          constants.Id()));
+        
+        columnMetas.add(new ColumnMeta<>(errorCountColumn, constants.Errors()));
```

Comment:
Not sure if I'm missing something but I still get the error column after the Id. I can see you changed it in here but I'm wondering if it could be related to this: https://github.com/rbarriuso/jbpm-wb/blob/e747572143dfa1a39fcd81ef45e0a60821354ea4/jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java#L121